ClinGen/clincoded
# Connects to #524. Replace deprecated react-tools and jstransform with Babel.

This PR consists of installing various Babel-related npm packages, removing the react-tools and jstransform npm packages, as well as updating other npm packages.

The following npm packages were added for `Babel` and its setup with `jest`, `browserify` and `react`:
_babel-core
babel-jest
babel-plugin-transform-runtime
babel-preset-es2015
babel-preset-jest
babel-preset-react
babel-polyfill
babel-runtime
babelify_

**Testing steps for #524:**
1. Git checkout the `524_jz_babel_conversion` branch and do a bin/buildout for the app.
2. Because switching to Babel from react-tools/JSTransform essentially impact all front-end code of the gene curation app, testing on all interface of the site is highly desirable.
3. In particular, the persona login needs to be ensured to work as expected.
4. For JavaScript testing, run `npm test` to make sure all tests pass.
